pkgs/applications/networking/cluster/tf-summarize/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, testers
+, tf-summarize
+}:
+
+buildGoModule rec {
+  pname = "tf-summarize";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "dineshba";
+    repo = "tf-summarize";
+    rev = "v${version}";
+    sha256 = "0c6fcz0n22mq8bqr82h9lfxx4n1bk9gjlc7d131lpf14yiacih3p";
+  };
+
+  vendorSha256 = "cnybdZth7qlP2BHK8uvLCoqJtggMIkvaL2+YugiUZRE=";
+  ldFlags = [ "-s -w -X main.version=${version}" ];
+
+  passthru.tests.version = testers.testVersion {
+    package = tf-summarize;
+    command = "tf-summarize -v";
+    version = "development";
```

Comment:
does `tf-summarize -v` print "development" instead of its actual package version?  If so that may suggest that your ldFlags passing main.version isn't having the intended effect

Reply:
Yes, we can see it here: https://github.com/dineshba/tf-summarize/blob/v0.3.2/main.go#L14

That makes sense, and I will update that.